var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -119,7 +119,26 @@ class Cp2k(MakefilePackage, CMakePackage, CudaPackage, ROCmPackage):
     variant("dftd4", when="@2024.2:", default=False, description="Enable DFT-D4 support")
     variant("mpi_f08", default=False, description="Use MPI F08 module")
     variant("smeagol", default=False, description="Enable libsmeagol support", when="@2025.2:")
-
+    variant(
+        "pw_gpu",
+        default=True,
+        description="Enable FFT calculations on GPU (off by default for now)",
+        when="@2025.2: +cuda",
+    )
+    variant("grid_gpu", default=True, description="Enable grid gpu backend", when="@2025.2:")
+    variant("dbm_gpu", default=True, description="Eanble DBM gpu backend", when="@2025.2:")
+    variant(
+        "pw_gpu",
+        default=False,
+        description="Enable FFT calculations on GPU",
+        when="@2025.2: +rocm",
+    )
+    variant(
+        "hip_backend",
+        default=False,
+        description="Enable HIP backend on Nvidia GPU",
+        when="@2025.2:+cuda",
```

Comment:
Minor, just for readability:
```suggestion
        when="@2025.2: +cuda",
```
?